src/components/chat/comment-textbox.tsx

```diff
@@ -26,7 +25,9 @@ export const CommentTextBox: React.FC<IProps> = (props) => {
   const [tagText, setTagText] = useState("");
   const textareaStyle = {height: commentTextAreaHeight};
   const postButtonClass = classNames("comment-footer-button", "themed-negative", activeNavTab,
-                                     { disabled: !commentAdded, "no-action": !commentAdded });
+                                      { disabled: (!commentAdded && !showCommentTag), //disabled if empty and no tags
```

Comment:
Before it was `disabled: (!commentAdded && !commentTags)` but since the curriculum didn't have commentTags - it was an empty object {} which is not falsy.

Reply:
Since `!commentAdded && !showCommentTag` is duplicated, you might want to save it in a variable with a descriptive name.